### ISSUE TITLE
Game Freezes When Loading “Mission 299: Sora”

### ISSUE THREAD

**Issue by joo7xxx** [vitor251093/KHMelonMix#245]
Completed mission 298 and closed out mission review. When Mission 299 screen loads and fades in thats where it gets stuck on a black screen.

**Comment by joo7xxx** [comment]
On steamdeck btw

**Comment by joo7xxx** [comment]
I believe I found the issue. Some HD cutscenes are missing in the assets folder that are not available in the zip file thats needed to download. This causes the game to freeze.

**Comment by vitor251093** [comment]
If it freezes on a black screen, with no sound, for an undetermined amount of time, I don't think this is related to the lack of a cutscene...

Not only all the cutscenes with DS equivalents are there (cutscenes that are in-game dialogs are really not present, but they aren't used either), but also, if any cutscene is missing, MM simply plays the DS cutscene.

It may be caused by a corrupted video file though.

**Comment by joo7xxx** [comment]
The way I managed to bypass it was by playing through day 299 on melonds and jumping back on melonmix for day 300 thereafter.

**Comment by vitor251093** [comment]
Does it freezes on a black screen, with no sound, for an undetermined amount of time? Or any of those didn't happen?

Also, does Day 299 starts with a prerendered cutscene (FMV) or with an in-game engine cutscene (text dialogs)?

**Comment by joo7xxx** [comment]
Correct, It freezes on a black screen with no sound right after the title, “Day 299: Sora,” fades out. Day 299 starts with an in game engine cutscene.

**Comment by vitor251093** [comment]
In that case it has no connection with the HD cutscene replacement. It is still a bug though.

Would you still have a save file from before 299 starts?

**Comment by joo7xxx** [comment]
I may be confusing fmv for an in engine cutscene. So i provided the video

**Comment by joo7xxx** [comment]
I believe i still do have the save file for that day.

**Comment by vitor251093** [comment]
Yeah, this is a FMV. Although, that cutscene is there. It's just that the file names are not in the order where they appear in the game, they are in the order where that appear inside the ROM data.

It's the hd846.mp4 file.

**Comment by vitor251093** [comment]
@joo7xxx can you confirm if you can still reproduce this bug on the latest Actions build (not the latest official release)?

**Comment by joo7xxx** [comment]
Is it possible to link me to the latest action build? Sorry, Im new to this.

**Comment by vitor251093** [comment]
Windows: https://github.com/vitor251093/KHMelonMix/actions/runs/10842381959

Steam Deck: https://github.com/vitor251093/KHMelonMix/actions/runs/10842381971

**Comment by joo7xxx** [comment]
Still run into the same issue.

**Comment by vitor251093** [comment]
Did you get to check if you have this file in your folder?

> It's the hd846.mp4 file, which is inside the MEGA folder.

**Comment by vitor251093** [comment]
Can you two confirm if the issue can still be reproduced with the new latest Actions build?

**Comment by joo7xxx** [comment]
I do have file hd846 and just tried the latest action build. Still have the same issue.
